models/product/src/product-variant/product-variant-draft/presets/sample-data-goodstore/square-bamboo-coaster-01.ts

```diff
@@ -0,0 +1,48 @@
+import { Money, PriceDraft } from '@commercetools-test-data/commons';
+import { AttributeDraft } from '../../../../attribute';
+import { ImageDraft } from '../../../../image';
+import type { TProductVariantDraftBuilder } from '../../../types';
+import * as ProductVariantDraft from '../../index';
+
+const squareBambooCoaster01 = (): TProductVariantDraftBuilder =>
+  ProductVariantDraft.presets
+    .empty()
+    .sku('Bamb-084')
```

Comment:
This is the first SKU I've seen that's lowercase

Reply:
Eagle eyes! Resolved in aa424d051a8abaf11bc62c2f5831503ab82305d8